### PR TITLE
Add Fast Healing effect to Field of Undeath auras

### LINKED ISSUE #7534: Deathless acolyte of ugathoa Fast healing incorrect
https://2e.aonprd.com/Monsters.aspx?ID=1853

when draging on the fast healing to a pc it does different amounts of fast healing.  it should be 2 not 5. which is whats on the re. 

image

### PR DESCRIPTION
**Pull request by stwlam**
Closes #7534

### CODE REVIEW

**Comment by simonward** [comment]
Please also update Seldrick Dralston (1-2) and Seldrick Dralston (3-4).

File: packs/data/book-of-the-dead-bestiary.db/deathless-acolyte-of-urgathoa.json [hunk 1]
```diff
@@ -1959,24 +1959,39 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>@Template[type:emanation|distance:20]{20 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>Allies within the aura who have negative healing gain @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Fast Healing]{Fast Healing 2}. When a creature within the aura would receive positive healing, it must succeed at a @Check[type:flat|dc:5] or receive no healing from the effect.</p>"
+                    "value": "<p>Allies within the aura who have negative healing gain fast healing 2. When a creature within the aura would receive positive healing, it must succeed at a @Check[type:flat|dc:5] or receive no healing from the effect.</p>"
```
Comment (simonward) [inline packs/data/book-of-the-dead-bestiary.db/deathless-acolyte-of-urgathoa.json:1962]: Generally we've left the template and aura links in place when adding aura REs. Should removing them be the standard going forward?

Comment (stwlam) [reply packs/data/book-of-the-dead-bestiary.db/deathless-acolyte-of-urgathoa.json:1962]: It's rather silly to include a template that serves no other purpose than to misdirect the user.

File: packs/data/book-of-the-dead-bestiary.db/deathless-hierophant-of-urgathoa.json [hunk 2]
```diff
@@ -2643,24 +2643,39 @@
...
+                                "includesSelf": false,
+                                "predicate": [
+                                    "target:negative-healing"
+                                ],
+                                "uuid": "Compendium.pf2e.bestiary-effects.x8e0MXrPgdRYQqBm"
+                            }
+                        ],
                         "key": "Aura",
-                        "radius": 40,
+                        "radius": 20,
```
Comment (simonward) [inline packs/data/book-of-the-dead-bestiary.db/deathless-hierophant-of-urgathoa.json:2670]:
```suggestion
                        "radius": 40,
```

Comment (stwlam) [reply packs/data/book-of-the-dead-bestiary.db/deathless-hierophant-of-urgathoa.json:2670]: Has anyone reported an issue to AoN?

File: packs/data/book-of-the-dead-bestiary.db/deathless-acolyte-of-urgathoa.json (hunk 1 above)
Comment (TikaelSol) [reply packs/data/book-of-the-dead-bestiary.db/deathless-acolyte-of-urgathoa.json:1962]: We can definitely revisit now that the aura RE exists, but we should have the aura size in the text as it is in the book.

**Pull request merged**